e2e_playwright/st_pydeck_chart_select_test.py

```diff
@@ -55,12 +56,30 @@
 STANDARD_WAIT_DELAY = 5000
 
 
+def _select_chart_type(app: Page, chart_type: str):
+    """Select the chart type to display."""
+    button_text_map = {
+        "basic": "Basic Chart",
+        "callback": "With Callback",
+        "form": "In Form",
+        "fragment": "In Fragment",
+        "scatterplot": "Scatterplot",
+    }
+
+    button_text = button_text_map.get(chart_type, "Basic Chart")
+    click_button(app, button_text)
+    wait_for_app_run(app, wait_delay=STANDARD_WAIT_DELAY)
```

Comment:
nit: I think `click_button` already waits for the app run, just doing an additional timeout is probably enough